### PR TITLE
Update release notes for V1.0.6

### PR DESCRIPTION
**Pull request by jeroenpf**
Related to pfHvTO-ar-p2

## Proposed Changes

I propose to add release notes for 1.0.6 release.

## Testing Instructions

1. Review release notes and check if all prominent changes are included

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
+1.0.6
+=====
+* Fixed update snapshot counter when site is deleted #344
+* Added support for Vietnamese language #348
+* Added restart nudge on check for update #353
+* Added hover state for site thumbnail #357
+* Delete site files checkbox checked by default #352
+* Improve loading indicator when adding new site #362 #367
```
Comment (wojtekn) [inline RELEASE-NOTES.txt]:
```suggestion
* Improved loading indicator when adding new site #362 #367
```

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+* Disable add site button while site is being created #371
```
Comment (wojtekn) [inline RELEASE-NOTES.txt]:
```suggestion
* Disabled add site button while site is being created #371
```

File: RELEASE-NOTES.txt [hunk 3]
```diff
@@ -1,3 +1,13 @@
+1.0.6
+=====
+* Fixed update snapshot counter when site is deleted #344
+* Added support for Vietnamese language #348
+* Added restart nudge on check for update #353
+* Added hover state for site thumbnail #357
+* Delete site files checkbox checked by default #352
```
Comment (wojtekn) [inline RELEASE-NOTES.txt]:
```suggestion
* Made 'Delete site files' checkbox checked by default #352
```

**Review by wojtekn — APPROVED** [review]
Looks good, I proposed to adjust some wording to match other entries style.

**Comment by kozer** [comment]
As discussed, let's also add https://github.com/Automattic/studio/pull/340.

**Review by kozer — APPROVED** [review]

**Pull request merged**